backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/invgroup/interceptor/ModCntrInvoicingGroupRepository.java

```diff
@@ -86,8 +86,8 @@ public Optional<ProductId> getInvoicingGroupProductFor(@NonNull final ProductId
 				.addOnlyActiveRecordsFilter()
 				.addEqualsFilter(I_ModCntr_InvoicingGroup_Product.COLUMNNAME_M_Product_ID, productId)
 				.andCollect(I_ModCntr_InvoicingGroup_Product.COLUMN_ModCntr_InvoicingGroup_ID)
-				.addCompareFilter(I_ModCntr_InvoicingGroup.COLUMNNAME_ValidFrom, Operator.LESS, Instant.now())
-				.addCompareFilter(I_ModCntr_InvoicingGroup.COLUMNNAME_ValidTo, Operator.GREATER, Instant.now())
+				.addCompareFilter(I_ModCntr_InvoicingGroup.COLUMNNAME_ValidFrom, Operator.LESS, SystemTime.asInstant())
```

Comment:
I think this should be lessOrEqual, because of `de.metas.contracts.modular.invgroup.interceptor.ModCntrInvoicingGroupRepository.validateInvoicingGroupProductNoOverlap`. With `addIntervalIntersection` an invoicingGroupProduct should be selected with validFrom included and validTo excluded.
Also this corner cases should be covered by cucumber.

Reply:
adjusting cucumber scenarios 👍